test/simple/simple.status

```diff
@@ -1 +1,35 @@
 prefix simple
+
+test-http-client-timeout-event          : PASS,FLAKY
```

Comment:
@orangemocha How did you gather that information? By looking at the history of failing tests for the nodejs-v0.10 Jenkins job on jenkins.nodejs.org?